locales/de/subtitles/vincent-pascucci/vincent-pascucci-02.yml

```diff
@@ -93,13 +90,12 @@ de:
           s: 17
           from: 00:01:50,19
           to: 00:02:08,09
-          text: The Communist Party was banned after the non-aggression treaty between
-            Hitler and Stalin (1939).
+          text: Das war nach dem Nicht-Angriffs-Pakt zwischen Hitler und Stalin (1939).
```

Comment:
@pixelpogo The English transcript is fine. I’ve now adapted the German translation to be closer to that.